### PR TITLE
docs: use side navigation instead of sections toolbar

### PR DESCRIPTION
**Pull request by chrisdavisp**
Issue Link: https://github.com/SAP/fundamental-ngx/issues/2481

### CODE REVIEW

**Comment by I322882** [comment]
Solved issue #2481

File: apps/docs/src/app/documentation/core-helpers/sections-toolbar/sections-toolbar.component.html [hunk 1]
```diff
@@ -12,26 +12,29 @@
...
-                                {{ item.name }}
-                            </a>
-                        </div>
-                    </li>
-                </ng-container>
-            </ul>
-        </nav>
+        <fd-side-nav>
+            <div fd-side-nav-main>
+                <ul fd-nested-list [textOnly]="true">
```
Comment (salarenko) [inline apps/docs/src/app/documentation/core-helpers/sections-toolbar/sections-toolbar.component.html:17]: Should't this be in reverse order?
`nav` -> `ul` -> `li`
I think you need to consult @JKMarkowski, he is preparing an update for side-nav and some markup might change.

**Comment by JKMarkowski** [comment]
Hi Chris, I would go for 
nav>ul>li(title: component, pattern etc)>ul>li(component name: combobox, datepicker etc)
So we will have [multiple level](https://fundamental-ngx.netlify.app/#/core/sideNavigation#withMultipleTitles) side nav. Let's wait a little bit until new markup for side nav is merged. I will let you know when it's ready

**Comment by stefanoScalzo** [comment]
In the meantime please change the title to follow semantic versioning please

**Comment by chrisdavisp** [comment]
closing and creating new pr with updated master

**Pull request closed without merging**